sources/packages/web/src/components/common/OfferingSummary.vue

```diff
@@ -4,28 +4,102 @@
     :records-count="offeringsAndCount.count"
   >
     <template #actions>
-      <v-row class="m-0 p-0">
-        <v-text-field
-          density="compact"
-          label="Search Offering Name"
-          variant="outlined"
-          v-model="searchBox"
-          data-cy="searchBox"
-          @keyup.enter="searchOfferingTable"
-          prepend-inner-icon="mdi-magnify"
-          hide-details="auto"
-        />
-        <v-btn
-          v-if="allowOfferingEdit"
-          class="ml-2 float-right"
-          @click="goToAddNewOffering()"
-          color="primary"
-          prepend-icon="fa:fa fa-plus-circle"
-          data-cy="addNewOfferingButton"
-        >
-          Add offering
-        </v-btn>
-      </v-row>
+      <v-form ref="searchOfferingsForm">
+        <div class="d-flex flex-wrap align-center ga-3">
+          <!-- Search Field -->
+          <v-text-field
+            density="compact"
+            label="Search Offering Name"
+            variant="outlined"
+            v-model="searchBox"
+            data-cy="searchBox"
+            @update:model-value="debouncedSearch"
+            @keyup.enter="searchOfferingTable"
+            prepend-inner-icon="mdi-magnify"
+            hide-details="auto"
+            class="flex-grow-1"
+          />
+
+          <!-- Date Range Group - stays together -->
+          <div class="d-flex align-center ga-2 flex-grow-1">
+            <v-date-input
+              density="compact"
+              variant="outlined"
+              label="From (Study Start Date)"
+              hide-details="auto"
+              prepend-icon=""
+              append-inner-icon="mdi-calendar"
+              v-model="startDate"
+              @update:model-value="debouncedSearch"
+            />
+            <v-date-input
+              density="compact"
+              variant="outlined"
+              label="To (Study Start Date)"
+              hide-details="auto"
+              prepend-icon=""
+              append-inner-icon="mdi-calendar"
+              v-model="endDate"
+              @update:model-value="debouncedSearch"
+            />
```

Comment:
Can we have the dates displayed in a date format accepted by business? ISO date format is followed by the form.io and it is used in many screens.

<img width="1420" height="98" alt="Image" src="https://github.com/user-attachments/assets/e7db8fc9-1748-4a8d-a833-672b0f0b948d" />